test-data/unit/fine-grained.test

```diff
@@ -5405,6 +5407,7 @@ T = int
 [out]
 ==
 main:4: error: "C" expects no type arguments, but 1 given
+main:4: error: "C" expects no type arguments, but 1 given
```

Comment:
whats up with these tests? `C` does take a type argument

Reply:
these tests are for the cache